src/i18n/utils.js

```diff
@@ -12,3 +12,11 @@ export function flattenMessages(nestedMessages, prefix = '') {
     return messages;
   }, {});
 }
+
+export const defaultLanguage = 'en';
+
+export const availableLanguageOrNull = (lang) =>
+  (lang && ['es', 'en'].includes(lang) && lang) || null;
+
+export const availableLanguageOrDefault = (lang) =>
+  availableLanguageOrNull(lang) || defaultLanguage;
```

Comment:
Why not have all in one function? `availableLanguageOrNull` as part of `availableLanguageOrDefault`

Reply:
Because in a place I need _null_ result. I did not analyze if it is really necessary (I think so) or not.

Reply:
You think you might need it, ok.

Reply:
Here: https://github.com/FromDoppler/doppler-webapp/pull/222/files#diff-14b1e33d5bf5649597cdc0e4f684daddR48